Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -72,17 +75,38 @@ FieldProbe::FieldProbe (std::string rd_name)
      *     Define whether ot not to integrate fields
      */
     amrex::ParmParse pp_rd_name(rd_name);
+    pp_rd_name.query("probe_geometry", probe_geometry);
+    if (probe_geometry == DetectorGeometry::Point)
+    {
+        getWithParser(pp_rd_name, "x_probe", x_probe);
+        getWithParser(pp_rd_name, "y_probe", y_probe);
+        getWithParser(pp_rd_name, "z_probe", z_probe);
+    }
+    else
```

Comment:
Rewrite this using [defensive programming](https://en.wikipedia.org/wiki/Defensive_programming):
`if(point)` - `else if(line)` - `else if(plane)` - `else { abort mission with a helpful error message }`

Reply:
@TiberiusRheaume reminder to change the logic here :)